### PR TITLE
build(npm): Dependency update (Semantic release to v20)

### PR DESCRIPTION
**Pull request by AsifNawaz-cnic**

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -48,12 +48,9 @@ RUN usermod --shell /usr/bin/zsh ${USERNAME} \
...
 
 # zsh and commitizen (cz) configurations
 COPY ${CONFIGURATION_DIR_PATH}/.zshrc ${USERHOME}/.zshrc
 COPY ${CONFIGURATION_DIR_PATH}/.czrc ${USERHOME}/.czrc
 RUN chown ${USERNAME}.${USERNAME} ${USERHOME}/.zshrc ${USERHOME}/.czrc \
-    && chmod +x ${USERHOME}/.zshrc ${USERHOME}/.czrc
-
-ENTRYPOINT exec top -b
\ No newline at end of file
+    && chmod +x ${USERHOME}/.zshrc ${USERHOME}/.czrc
```
Comment (KaiSchwarz-cnic) [inline .devcontainer/Dockerfile:56]: if you do `chmod +x` locally and add + commit the change, this line is superfluous.

**Review by KaiSchwarz-cnic — CHANGES_REQUESTED** [review]
thanks for reviewing!

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (AsifNawaz-cnic) [reply .devcontainer/Dockerfile]: Thanks for your feedback, done making the changes.

**Review by KaiSchwarz-cnic — APPROVED** [review]
ty

**Pull request merged**